[glew] Unable to build

```
  It fails with the following output:

    Change Dir: /usr/local/tmp/crew/download.dir/glew-2.2.0/build/CMakeFiles/CMakeTmp
    
    Run Build Command(s):/usr/local/bin/make cmTC_810de/fast && /usr/local/bin/make  -f CMakeFiles/cmTC_810de.dir/build.make CMakeFiles/cmTC_810de.dir/build
    make[1]: Entering directory '/usr/local/tmp/crew/download.dir/glew-2.2.0/build/CMakeFiles/CMakeTmp'
    Building C object CMakeFiles/cmTC_810de.dir/testCCompiler.c.o
    /usr/local/bin/cc   -fuse-ld=lld  -o CMakeFiles/cmTC_810de.dir/testCCompiler.c.o -c /usr/local/tmp/crew/download.dir/glew-2.2.0/build/CMakeFiles/CMakeTmp/testCCompiler.c
    cc: error: unrecognized command line option ‘-fuse-ld=lld’; did you mean ‘-fuse-ld=bfd’?
    make[1]: *** [CMakeFiles/cmTC_810de.dir/build.make:85: CMakeFiles/cmTC_810de.dir/testCCompiler.c.o] Error 1
    make[1]: Leaving directory '/usr/local/tmp/crew/download.dir/glew-2.2.0/build/CMakeFiles/CMakeTmp'
    make: *** [Makefile:140: cmTC_810de/fast] Error 2
    
    

  

  CMake will not be able to correctly generate this project.
Call Stack (most recent call first):
  CMakeLists.txt:5 (project)


-- Configuring incomplete, errors occurred!
See also "/usr/local/tmp/crew/download.dir/glew-2.2.0/build/CMakeFiles/CMakeOutput.log".
See also "/usr/local/tmp/crew/download.dir/glew-2.2.0/build/CMakeFiles/CMakeError.log".
glew failed to build: `cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr/local -DCMAKE_INSTALL_LIBDIR=/usr/local/lib ./cmake` exited with 1
```

--- a/packages/gcalculator.rb
+++ b/packages/gcalculator.rb
@@ -6,13 +6,19 @@ class Gcalculator < Package
   version '3.38.1'
   compatibility 'all'
   source_url 'https://gitlab.gnome.org/GNOME/gnome-calculator/-/archive/3.38.1/gnome-calculator-3.38.1.tar.bz2'
-  source_sha256 '07dc348eef03f81fbe9c99f30836e0a1438942e5a9e79655772414eb5b09b691'
+  source_sha256 '33706fe3ade10b58e39c008f005a5bcca1b59239047b8c8f3ed595d0b7bd6894'
 
   binary_url ({
-    
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gcalculator-3.38.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gcalculator-3.38.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gcalculator-3.38.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gcalculator-3.38.1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    
+    aarch64: '1ab4852cb55661918bc646bf6b1ca1ecf153c3fcc78d46cf13b1e7bc830ee69d',
+     armv7l: '1ab4852cb55661918bc646bf6b1ca1ecf153c3fcc78d46cf13b1e7bc830ee69d',
+       i686: '0eb813d120d974365d2910e9a21bee9bdb6e533b0d21e26510b40cbe09d3c4ca',
+     x86_64: '8b1478a211443794b6aba531b30ff9359080cb580e5214d27d8df97beac3c0ec',
   })
 
   depends_on 'setuptools' => :build

--- a/packages/glew.rb
+++ b/packages/glew.rb
@@ -8,9 +8,20 @@ class Glew < Package
   source_url 'https://sourceforge.net/projects/glew/files/glew/snapshots/glew-20200115.tgz/download'
   source_sha256 '314219ba1db50d49b99705e8eb00e83b230ee7e2135289a00b5b570e4a4db43a'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/glew-2.2.0-rc3-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/glew-2.2.0-rc3-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/glew-2.2.0-rc3-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/glew-2.2.0-rc3-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '0912816e2c59d5a677f0b777ac92b7ac92262a9bf8daa5ab5c9e182b95cb9836',
+     armv7l: '0912816e2c59d5a677f0b777ac92b7ac92262a9bf8daa5ab5c9e182b95cb9836',
+       i686: 'f2b24b7826fe5c8b46a333ae7ece33a1d3a6a73f2f499c769a8a22fae4d7b53e',
+     x86_64: '3c79783cf39d26b793fed5b0b9d655febda3ec57336cdacd597f574fc29253bb',
+  })
 
   depends_on 'mesa'
-  depends_on 'llvm' => :build
 
   def self.patch
     system 'tar fxv ../download'
@@ -22,9 +33,9 @@ class Glew < Package
   end
   
   def self.build
-    ENV['CFLAGS.EXTRA'] = "-fuse-ld=lld"
-    ENV['CFLAGS'] = "-fuse-ld=lld"
-    ENV['CXXFLAGS'] = "-fuse-ld=lld"
+    #ENV['CFLAGS.EXTRA'] = "-fuse-ld=lld"
+    #ENV['CFLAGS'] = "-fuse-ld=lld"
+    #ENV['CXXFLAGS'] = "-fuse-ld=lld"
     ENV['GLEW_DEST'] = "#{CREW_PREFIX}"
     Dir.chdir 'glew-2.2.0/build' do
       system 'cmake',

--- a/packages/gobject_introspection.rb
+++ b/packages/gobject_introspection.rb
@@ -8,6 +8,19 @@ class Gobject_introspection < Package
   source_url 'https://download.gnome.org/sources/gobject-introspection/1.66/gobject-introspection-1.66.1.tar.xz'
   source_sha256 'dd44a55ee5f426ea22b6b89624708f9e8d53f5cc94e5485c15c87cb30e06161d'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gobject_introspection-1.66.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gobject_introspection-1.66.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gobject_introspection-1.66.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gobject_introspection-1.66.1-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '32ff02babe0624b822bc71f0405d6db9babc276af4d24660af3eb703f9d7959c',
+     armv7l: '32ff02babe0624b822bc71f0405d6db9babc276af4d24660af3eb703f9d7959c',
+       i686: 'e63d47a08feefa228e260fd3dd13cae6f85f38c96942b487abdf3f6a6076bc13',
+     x86_64: 'adb621c7edcc29dc653140bd07260270eead0ca8fb26e063ff9788c098c7dfab',
+  })
+
   depends_on 'glib'
   depends_on 'cairo'
 

--- a/packages/gsl.rb
+++ b/packages/gsl.rb
@@ -8,16 +8,28 @@ class Gsl < Package
   source_url 'https://ftpmirror.gnu.org/gsl/gsl-2.6.tar.gz'
   source_sha256 'b782339fc7a38fe17689cb39966c4d821236c28018b6593ddb6fd59ee40786a8'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gsl-2.6-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gsl-2.6-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gsl-2.6-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gsl-2.6-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'aef58137cae2223ebc52ee81039e04768d32c837edbbddcc25d144cdd0e1983b',
+     armv7l: 'aef58137cae2223ebc52ee81039e04768d32c837edbbddcc25d144cdd0e1983b',
+       i686: 'ff59679e039167ed10758575b49780f688d0e70dd5ee9bcc45d3546bae168f46',
+     x86_64: 'b08296445edc313d9c44a66a73a4a62c264c828fb8ee41e9d42d819742cee4f4',
+  })
 
   def self.build
     system './configure',
            "--prefix=#{CREW_PREFIX}",
            "--libdir=#{CREW_LIB_PREFIX}",
            '--disable-maintainer-mode'
-    system "make"
+    system 'make'
   end
 
   def self.install
-    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
   end
 end

--- a/packages/gst_plugins_base.rb
+++ b/packages/gst_plugins_base.rb
@@ -8,6 +8,18 @@ class Gst_plugins_base < Package
   source_url 'https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-1.18.0.tar.xz'
   source_sha256 '762abdd1a950809a1cea62fff7f86b5f7d6bd5f6841e3e585c700b823cdb7897'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gst_plugins_base-1.18.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gst_plugins_base-1.18.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gst_plugins_base-1.18.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gst_plugins_base-1.18.0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'a53d2597a9779c17677a600ac0cd669a8ce3079ac62ab87627880f2148fab970',
+     armv7l: 'a53d2597a9779c17677a600ac0cd669a8ce3079ac62ab87627880f2148fab970',
+       i686: 'b2ac2d8f40199531e304c35659ecbd48107297072114967011feb8b12a48ebba',
+     x86_64: 'f2ea971bba4cf271d4f36b7eb743065cc53ce00973afebb1076d996ff4e04170',
+  })
 
   depends_on 'gstreamer'
   depends_on 'libtheora'
@@ -25,12 +37,11 @@ class Gst_plugins_base < Package
   depends_on 'libglu'
   depends_on 'libgudev'
   depends_on 'gdk_pixbuf'
-  depends_on 'llvm' => :build
 
   def self.build
     # Use lld to enable compatibility with ChromeOS libraries
-    ENV['CFLAGS'] = "-fuse-ld=lld"
-    ENV['CXXFLAGS'] = "-fuse-ld=lld"
+    #ENV['CFLAGS'] = "-fuse-ld=lld"
+    #ENV['CXXFLAGS'] = "-fuse-ld=lld"
     system "meson",
     "--prefix=#{CREW_PREFIX}",
     "--libdir=#{CREW_LIB_PREFIX}",

--- a/packages/gstreamer.rb
+++ b/packages/gstreamer.rb
@@ -8,6 +8,19 @@ class Gstreamer < Package
   source_url 'https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.18.0.tar.xz'
   source_sha256 '0ff09245b06c0aeb5d9a156edcab088a7e8213a0bf9c84a1ff0318f9c00c7805'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gstreamer-1.18.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gstreamer-1.18.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gstreamer-1.18.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gstreamer-1.18.0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '0dca199353177541752ce291d76979019b80af95dfa7c9499fdb992010a19151',
+     armv7l: '0dca199353177541752ce291d76979019b80af95dfa7c9499fdb992010a19151',
+       i686: '5de8acc27668a03cbf674589bbb840848212afa265657ef0054f2c037ab08702',
+     x86_64: '2a9044b75cf3d305675486a951054cf792495f7baeb38eab27fa1aa5f6b50bf7',
+  })
+
   depends_on 'glib'
   depends_on 'libcap'
   depends_on 'gtk3'
@@ -15,12 +28,11 @@ class Gstreamer < Package
   depends_on 'elfutils'
   depends_on 'libunwind'
   depends_on 'python27'
-  depends_on 'llvm' => :build
 
   def self.build
     # Use lld for ChromOS library compatibility
-    ENV['CFLAGS'] = "-fuse-ld=lld"
-    ENV['CXXFLAGS'] = "-fuse-ld=lld"
+    #ENV['CFLAGS'] = "-fuse-ld=lld"
+    #ENV['CXXFLAGS'] = "-fuse-ld=lld"
     system "meson",
     "--prefix=#{CREW_PREFIX}",
     "--libdir=#{CREW_LIB_PREFIX}",

--- a/packages/gtk3.rb
+++ b/packages/gtk3.rb
@@ -8,6 +8,19 @@ class Gtk3 < Package
   source_url 'https://download.gnome.org/sources/gtk+/3.24/gtk+-3.24.23.tar.xz'
   source_sha256 '5d864d248357a2251545b3387b35942de5f66e4c66013f0962eb5cb6f8dae2b1'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gtk3-3.24.23-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gtk3-3.24.23-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gtk3-3.24.23-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gtk3-3.24.23-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'dcb34ab52e30687e837381fa223d0935d89622d380e8187b9de2db87d0cc49fd',
+     armv7l: 'dcb34ab52e30687e837381fa223d0935d89622d380e8187b9de2db87d0cc49fd',
+       i686: '2a835418e1553be488b2f3e9dddda16bb2e90cb5b1b77abead42f0b79bb73ec6',
+     x86_64: '4db8850af34d796ad8460a96f1b5c079d52b8c085833f2b39ed31c688db1e964',
+  })
+
   depends_on 'gdk_pixbuf'
   depends_on 'iso_codes'
   depends_on 'json_glib'

--- a/packages/heroku.rb
+++ b/packages/heroku.rb
@@ -14,9 +14,9 @@ class Heroku < Package
      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/heroku-7.44.0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: '60c419195c5a9c40b9bccc18d9da7d50a122e6609784a85c3fd5ea9f882bcf26',
-     armv7l: '60c419195c5a9c40b9bccc18d9da7d50a122e6609784a85c3fd5ea9f882bcf26',
-     x86_64: 'fa9b538dd683248d3e892f8a74c16c2a124dc189e4b9b0b18c17e57897b93096',
+    aarch64: 'c97d289b8e06b35552f49c2da468a883292a8bd29f40d5bf110b0e3632cfc102',
+     armv7l: 'c97d289b8e06b35552f49c2da468a883292a8bd29f40d5bf110b0e3632cfc102',
+     x86_64: '2e98fdebcbe61a6819e7066da0152a1da1cabaa456d4d6f869d4ec582b5e8555',
   })
 
   depends_on 'yarn' => :build

--- a/packages/jfrog.rb
+++ b/packages/jfrog.rb
@@ -9,20 +9,26 @@ class Jfrog < Package
   source_sha256 '75a0f9bfd3170d03c542b62442761822ff2e4a7a279406d9b2ee5eeee888813f'
 
   binary_url ({
-    
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/jfrog-1.39.5-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/jfrog-1.39.5-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/jfrog-1.39.5-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/jfrog-1.39.5-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    
+    aarch64: '4561d7c7fe855a92055e23baee586f25b0261fce20f89919c12ca477e0895324',
+     armv7l: '4561d7c7fe855a92055e23baee586f25b0261fce20f89919c12ca477e0895324',
+       i686: 'a9e442aaa7c27b21256b8fc43d8f14b97ab3b65efa247812f5fb20ed57cbf028',
+     x86_64: '260c9a12692b2942cba44748dd5bdc72efbf377064ddd242db4100bff4a175ca',
   })
 
   depends_on 'go' => :build
 
   def self.build
     system "git clone https://github.com/jfrog/jfrog-cli-go.git"
-    system "cd jfrog-cli-go/jfrog-cli/jfrog/ && git checkout #{version} && go build"
+    system "cd jfrog-cli-go && git checkout #{version} && build/build.sh"
   end
 
   def self.install
-    system "install -Dm755 jfrog-cli-go/jfrog-cli/jfrog/jfrog #{CREW_DEST_PREFIX}/bin/jfrog"
+    system "install -Dm755 jfrog-cli-go/jfrog #{CREW_DEST_PREFIX}/bin/jfrog"
   end
 end

--- a/packages/keybase.rb
+++ b/packages/keybase.rb
@@ -6,16 +6,22 @@ class Keybase < Package
   version '5.5.1'
   compatibility 'all'
   source_url 'https://github.com/keybase/client/archive/v5.5.1.tar.gz'
-  source_sha256 'a65dc4b62fc1299dd17da52ddd2484fa1dc1s7d2a4776c3a6e112ee020980b12'
+  source_sha256 'a65dc4b62fc1299dd17da52ddd2484fa1dc1e7d2a4776c3a6e112ee020980b12'
 
   binary_url ({
-    
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/keybase-5.5.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/keybase-5.5.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/keybase-5.5.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/keybase-5.5.1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    
+    aarch64: '248fe6da5cd0a7c8e4663901a6acbcb94ca1347291309545e3e068aad6a0dd2d',
+     armv7l: '248fe6da5cd0a7c8e4663901a6acbcb94ca1347291309545e3e068aad6a0dd2d',
+       i686: '15662482b0381d8a3053897f21059bd425f87e06cd907c7ceb3e8f6e453f2afb',
+     x86_64: '554a661f752277920cb572de6aad58d0888e188defc0a4fb79ed8a0a3d8eadb2',
   })
 
-  depends_on 'go'
+  depends_on 'go' => :build
 
   def self.install
     system "go get github.com/keybase/client/go/keybase"

--- a/packages/libdrm.rb
+++ b/packages/libdrm.rb
@@ -8,6 +8,18 @@ class Libdrm < Package
   source_url 'https://dri.freedesktop.org/libdrm/libdrm-2.4.102.tar.xz'
   source_sha256 '8bcbf9336c28e393d76c1f16d7e79e394a7fce8a2e929d52d3ad7ad8525ba05b'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libdrm-2.4.102-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libdrm-2.4.102-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libdrm-2.4.102-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libdrm-2.4.102-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'd02b90125434ef5bd9f1bd86514929edc09a08089792cafdfaf2b678d3dc2ed5',
+     armv7l: 'd02b90125434ef5bd9f1bd86514929edc09a08089792cafdfaf2b678d3dc2ed5',
+       i686: '9d0cd25cdc0a2193ef30b1549d6d2979c94dd941a95c7a07f5ed84413da89cbe',
+     x86_64: '5f67dd18f9bf9653addbf2989b9179fa9c3d55fe855a3b1681243239a32987c4',
+  })
 
   depends_on 'libpciaccess'
   depends_on 'xorg_lib'

--- a/packages/libfilezilla.rb
+++ b/packages/libfilezilla.rb
@@ -9,8 +9,16 @@ class Libfilezilla < Package
   source_sha256 'fdd5b4ee1916af7b1d402e7dd467f05df7624bb65fb95a0875a03712ecc2d6fd'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libfilezilla-0.24.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libfilezilla-0.24.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libfilezilla-0.24.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libfilezilla-0.24.1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '1dbede14eac9364679565683f4473cb11996bf126b2317f8e707ff00337481ea',
+     armv7l: '1dbede14eac9364679565683f4473cb11996bf126b2317f8e707ff00337481ea',
+       i686: 'a105e9a6c565aad505e85cdf0618a272cd4d4efde09fb20fed768172bc8f841a',
+     x86_64: '7e61216d322cf0acc5c564cfc5b83663b8593ec420f22410943572ebde1f2aea',
   })
 
   def self.build

--- a/packages/libnftnl.rb
+++ b/packages/libnftnl.rb
@@ -8,12 +8,27 @@ class Libnftnl < Package
   source_url 'https://netfilter.org/projects/libnftnl/files/libnftnl-1.1.7.tar.bz2'
   source_sha256 '20dbc13f11004aea2c9e479cfb90359cb11fe3446c3140811c18e4ec1648ed8f'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libnftnl-1.1.7-1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libnftnl-1.1.7-1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libnftnl-1.1.7-1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libnftnl-1.1.7-1-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '6cc802f2d8a84367d34fc91c32aa8f06715a0cac91d4c6ea774fe2905be1255a',
+     armv7l: '6cc802f2d8a84367d34fc91c32aa8f06715a0cac91d4c6ea774fe2905be1255a',
+       i686: '510a3ca438c83f53457707bc89e6ddab8332fb7e5343721def87cbc9a86c34da',
+     x86_64: '7e72516f66aab3d3ed6d554e72c0b198ec82ee8e061a7c3941d607890de100f2',
+  })
+
+  depends_on 'libmnl'
+
   def self.build
-    system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
-    system "make"
+    system "./configure #{CREW_OPTIONS}"
+    system 'make'
   end
 
   def self.install
-    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
   end
 end

--- a/packages/libwebp.rb
+++ b/packages/libwebp.rb
@@ -8,6 +8,19 @@ class Libwebp < Package
   source_url 'https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-1.1.0.tar.gz'
   source_sha256 '98a052268cc4d5ece27f76572a7f50293f439c17a98e67c4ea0c7ed6f50ef043'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libwebp-1.1.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libwebp-1.1.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libwebp-1.1.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libwebp-1.1.0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '68c849400666f459e7c8e521a2cd1bb0303acbfdb7ffec9e07c57b24b7262555',
+     armv7l: '68c849400666f459e7c8e521a2cd1bb0303acbfdb7ffec9e07c57b24b7262555',
+       i686: '95ded30a5a8f9a448a2d02f528a207a1ef70417b41f1d3087f3f9b9eac78574e',
+     x86_64: '3fc837c1a69cc0970450b60644b2aa977cf061dd4379a4250509e08aa4ba0c71',
+  })
+
   depends_on 'libpng'
   depends_on 'libsdl'
   depends_on 'libtiff'

--- a/packages/lynis.rb
+++ b/packages/lynis.rb
@@ -3,31 +3,36 @@ require 'package'
 class Lynis < Package
   description 'Lynis is a security auditing tool for UNIX derivatives like Linux, macOS, BSD, Solaris, AIX, and others.'
   homepage 'https://cisofy.com/download/lynis/'
-  version '3.0.0'
+  version '3.0.1'
   compatibility 'all'
-  source_url 'https://cisofy.com/files/lynis-3.0.0.tar.gz'
-  source_sha256 '7da40f8c52d43b19eb9dc66aaece9c756c2ec581609ffce1a2c510cd99af38e1'
+  source_url 'https://downloads.cisofy.com/lynis/lynis-3.0.1.tar.gz'
+  source_sha256 '8381b62e11a5e0ead417bcfd92845adab7dc3b9d06271c852a1166cb65a61aff'
 
   binary_url ({
-    
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/lynis-3.0.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/lynis-3.0.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/lynis-3.0.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/lynis-3.0.1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    
+    aarch64: '4172f481e0e7fec75b0c5e7552a3cd6f22a4f6aed883bf8c08d3670216617f6b',
+     armv7l: '4172f481e0e7fec75b0c5e7552a3cd6f22a4f6aed883bf8c08d3670216617f6b',
+       i686: '9e3335b473b1b978c5c237a5866471220bf7024287c44735249be7a2d9f2094d',
+     x86_64: '73aa5996210bb14aca743a9a58bb8169fb377c1e9b19ac1e542484b11a86f260',
   })
 
   def self.install
-    system "mkdir -p #{CREW_DEST_PREFIX}/bin"
-    system "mkdir -p #{CREW_DEST_PREFIX}/man/man8"
-    system "mkdir -p #{CREW_DEST_PREFIX}/share/lynis"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/man/man8"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/lynis"
     system "gzip -9 lynis.8"
-    system "mv lynis.8.gz #{CREW_DEST_PREFIX}/man/man8"
-    system "cp -r . #{CREW_DEST_PREFIX}/share/lynis"
+    FileUtils.cp_r '.', "#{CREW_DEST_PREFIX}/share/lynis"
     system "echo '#!/bin/bash' > lynis"
     system "echo 'PWD=\$(pwd)' >> lynis"
     system "echo 'cd #{CREW_PREFIX}/share/lynis' >> lynis"
     system "echo './lynis \"\$@\"' >> lynis"
     system "echo 'cd \$PWD' >> lynis"
-    system "chmod +x lynis"
-    system "cp lynis #{CREW_DEST_PREFIX}/bin"
+    system "install -Dm755 lynis #{CREW_DEST_PREFIX}/bin/lynis"
+    system "install -Dm644 lynis.8.gz #{CREW_DEST_PREFIX}/man/man8/lynis.8.gz"
   end
 end

--- a/packages/mesa.rb
+++ b/packages/mesa.rb
@@ -8,6 +8,18 @@ class Mesa < Package
   source_url 'https://mesa.freedesktop.org/archive/mesa-20.2.0.tar.xz'
   source_sha256 '63f0359575d558ef98dd78adffc0df4c66b76964ebf603b778b7004964191d30'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/mesa-20.2.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/mesa-20.2.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/mesa-20.2.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/mesa-20.2.0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '48ea7803900737f3cb039916a194803df7c74350856baf1e32867f4c7dbaffc4',
+     armv7l: '48ea7803900737f3cb039916a194803df7c74350856baf1e32867f4c7dbaffc4',
+       i686: '09e41a089c50399f2b5667d19214df84e7542c1e35a9f2a48e3b1d1c844df69e',
+     x86_64: '8dc082dc17b1ba23d3e6590b7696dbdfcb63ed5a03fb22c7682ecd6fce529642',
+  })
 
   depends_on 'setuptools' => :build
   depends_on 'libva'

--- a/packages/mesa_utils.rb
+++ b/packages/mesa_utils.rb
@@ -8,16 +8,27 @@ class Mesa_utils< Package
   source_url 'https://mesa.freedesktop.org/archive/demos/mesa-demos-8.4.0.tar.bz2'
   source_sha256 '01e99c94a0184e63e796728af89bfac559795fb2a0d6f506fa900455ca5fff7d'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/mesa_utils-8.4.0-1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/mesa_utils-8.4.0-1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/mesa_utils-8.4.0-1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/mesa_utils-8.4.0-1-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '79f7aa0c1e8b69ab4b12045e2362ee75c98e700e01a3808a74c0f7d6aef4f74b',
+     armv7l: '79f7aa0c1e8b69ab4b12045e2362ee75c98e700e01a3808a74c0f7d6aef4f74b',
+       i686: '03db825ab275db23372dca3e76cb9676f4de8b0be232f96a15f5c500ee7b438a',
+     x86_64: 'c9f1bd525f7ace061488cc1780928f9e6face4509210bd5c348ad60dee2c97c9',
+  })
+
   depends_on 'mesa'
   depends_on 'glew'
-  depends_on 'llvm' => :build
+  depends_on 'freeglut'
 
   def self.build
-    ENV['CFLAGS'] = "-fuse-ld=lld"
-    ENV['CXXFLAGS'] = "-fuse-ld=lld"
-    system './configure',
-           "--prefix=#{CREW_PREFIX}",
-           "--libdir=#{CREW_LIB_PREFIX}"
+    #ENV['CFLAGS'] = "-fuse-ld=lld"
+    #ENV['CXXFLAGS'] = "-fuse-ld=lld"
+    system "./configure #{CREW_OPTIONS}"
     system 'make'
   end
 

--- a/packages/nftables.rb
+++ b/packages/nftables.rb
@@ -8,6 +8,19 @@ class Nftables < Package
   source_url 'https://netfilter.org/projects/nftables/files/nftables-0.9.6.tar.bz2'
   source_sha256 '68d6fdfe8ab02303e6b1f13968a4022da5b0120110eaee3233d806857937b66e'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/nftables-0.9.6-1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/nftables-0.9.6-1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/nftables-0.9.6-1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/nftables-0.9.6-1-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '87b2be9b1f371c684d5d21c111e8baf96f61813963ee38726c0fde7dd089eb47',
+     armv7l: '87b2be9b1f371c684d5d21c111e8baf96f61813963ee38726c0fde7dd089eb47',
+       i686: '08f6d4bd5f6f7e3a3b4dea10fd5cabd96001ac72e85c11757eff818a5eea1ea7',
+     x86_64: 'af2c671949dde45656db452e635af6f04e66b39c370ae0807afee0ad63a1e978',
+  })
+
   depends_on 'help2man'
   depends_on 'jansson'
   depends_on 'libmnl'

--- a/packages/openal.rb
+++ b/packages/openal.rb
@@ -9,9 +9,17 @@ class Openal < Package
   source_sha256 'c32d10473457a8b545aab50070fe84be2b5b041e1f2099012777ee6be0057c13'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/openal-1.20.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/openal-1.20.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/openal-1.20.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/openal-1.20.1-chromeos-x86_64.tar.xz',
     
   })
   binary_sha256 ({
+    aarch64: 'f4fad76fa80a35e5ed2b551623610a764942c599caa606474bb9585da30dd648',
+     armv7l: 'f4fad76fa80a35e5ed2b551623610a764942c599caa606474bb9585da30dd648',
+       i686: '34754828e5fdf5632b1a72f5632568fd3484b1edca17d14692f7bfa5578a0752',
+     x86_64: 'ad3c151c3cd255fac16bd21bf207ebcc14928d740691fc74ee5d5537627bb2d7',
    
   })
 

--- a/packages/openssh.rb
+++ b/packages/openssh.rb
@@ -8,6 +8,18 @@ class Openssh < Package
   source_url 'https://github.com/openssh/openssh-portable/archive/V_8_4_P1.tar.gz'
   source_sha256 'b924181983a77cb10e61530960e818ccae075e5c457ea6b9f67e4946009563db'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/openssh-8.4-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/openssh-8.4-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/openssh-8.4-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/openssh-8.4-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'c5b8da68a42483a6e5ffc032a7e699be340169fee6abe956c0b865b7ed617286',
+     armv7l: 'c5b8da68a42483a6e5ffc032a7e699be340169fee6abe956c0b865b7ed617286',
+       i686: '852034874b0fc63c61eb7db3a770bcf2fbec471153829c61d0f67794d102c5d6',
+     x86_64: '38aa21b22cde3bb2d52ee6c3826f92f83326985e58ceae9ef9aa9dcacb7de285',
+  })
 
   depends_on 'autoconf' => :build
   depends_on 'compressdoc' => :build

--- a/packages/pipewire.rb
+++ b/packages/pipewire.rb
@@ -4,29 +4,42 @@ class Pipewire < Package
   description 'PipeWire is a project that aims to greatly improve handling of audio and video under Linux.'
   homepage 'https://pipwire.org'
   version '0.3.13'
-  compatibility 'all'
+  compatibility 'aarch64,armv7l,x86_64'
   source_url 'https://gitlab.freedesktop.org/pipewire/pipewire/-/archive/0.3.13/pipewire-0.3.13.tar.gz'
   source_sha256 '40f2db86d8ba14706bfab8ee7c1789aeeb72bee386d1e44dbcd98888ef9861e5'
 
-  depends_on 'gsettings_desktop_schemas'
-  depends_on 'alsa_plugins' => :build
-  depends_on 'gst_plugins_base'
-  depends_on 'gstreamer'
-  depends_on "jack"
-  depends_on 'llvm' => :build
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/pipewire-0.3.13-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/pipewire-0.3.13-chromeos-armv7l.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/pipewire-0.3.13-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '3bc0dc8a59bd16789b238f9e093660ab4d81ba7e821a8d83bb755cc5cb528ce2',
+     armv7l: '3bc0dc8a59bd16789b238f9e093660ab4d81ba7e821a8d83bb755cc5cb528ce2',
+     x86_64: '259eedde6d48d78c87528ee31be78b55debb254e9cd40f662f192e608f000c1b',
+  })
+
+  case ARCH
+  when 'aarch64', 'armv7l', 'x86_64'
+    depends_on 'gsettings_desktop_schemas'
+    depends_on 'alsa_plugins' => :build
+    depends_on 'gst_plugins_base'
+    depends_on 'gstreamer'
+    depends_on 'jack'
+  end
 
   def self.build
-     ENV['CFLAGS'] = "-fuse-ld=lld"
-     ENV['CXXFLAGS'] = "-fuse-ld=lld"
+     #ENV['CFLAGS'] = "-fuse-ld=lld"
+     #ENV['CXXFLAGS'] = "-fuse-ld=lld"
      system "meson",
      "--prefix=#{CREW_PREFIX}",
      "--libdir=#{CREW_LIB_PREFIX}",
-      "-Dbluez5=false",
-      "-Dbluez5-backend-native=false",
-      "-Dbluez5-backend-ofono=false",
-      "-Dbluez5-backend-hsphfpd=false",
-      "-Dvulkan=false",
-      "build"
+     '-Dbluez5=false',
+     '-Dbluez5-backend-native=false',
+     '-Dbluez5-backend-ofono=false',
+     '-Dbluez5-backend-hsphfpd=false',
+     '-Dvulkan=false',
+     'build'
   end
 
   def self.install

--- a/packages/pmd.rb
+++ b/packages/pmd.rb
@@ -5,22 +5,28 @@ class Pmd < Package
   homepage 'https://pmd.github.io/'
   version '6.28.0'
   compatibility 'all'
-  source_url 'https://github.com/pmd/pmd/releases/download/pmd_releases%2F5.8.1/pmd-bin-6.28.0.zip'
-  source_sha256 '71912ee24f28b3607f6b3aabfc7cab13bbff84d43a319a78a05f0eca7c4ba4dc'
+  source_url 'https://github.com/pmd/pmd/releases/download/pmd_releases%2F6.28.0/pmd-bin-6.28.0.zip'
+  source_sha256 '9a19365f2e107ae801b39be04c5c03cdca2d352c450faac639a6dd95b5c3ab0c'
 
   binary_url ({
-    
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/pmd-6.28.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/pmd-6.28.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/pmd-6.28.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/pmd-6.28.0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    
+    aarch64: '3ffa1b5c562929b74903a42bb8246203b6b1a09a156031e25522991bb25eea51',
+     armv7l: '3ffa1b5c562929b74903a42bb8246203b6b1a09a156031e25522991bb25eea51',
+       i686: '8002bc791ac2751dca2c557b248a3d52eff48c5ad04d8ad4320fc4b19a88a230',
+     x86_64: 'a9595a5dc740d9f952526817488a86bca488ec06daac8b990b724eca45459629',
   })
 
   depends_on 'jdk8'
   depends_on 'unzip'
 
   def self.install
-    system "mkdir -p #{CREW_DEST_PREFIX}/bin"
-    FileUtils.cd("#{CREW_DEST_PREFIX}/bin") do
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
+    FileUtils.cd "#{CREW_DEST_PREFIX}/bin" do
       system "echo '#!/bin/bash' > cpd"
       system "echo 'PWD=$(pwd)' >> cpd"
       system "echo 'cd #{CREW_LIB_PREFIX}/pmd' >> cpd"
@@ -34,7 +40,7 @@ class Pmd < Package
       system "echo 'cd $PWD' >> pmd"
       system "chmod +x pmd"
     end
-    system "mkdir -p #{CREW_DEST_LIB_PREFIX}/pmd"
-    system "cp -r . #{CREW_DEST_LIB_PREFIX}/pmd"
+    FileUtils.mkdir_p "#{CREW_DEST_LIB_PREFIX}/pmd"
+    FileUtils.cp_r '.', "#{CREW_DEST_LIB_PREFIX}/pmd"
   end
 end

--- a/packages/sommelier.rb
+++ b/packages/sommelier.rb
@@ -3,11 +3,23 @@ require 'package'
 class Sommelier < Package
   description 'Sommelier works by redirecting X11 and Wayland programs to the built-in ChromeOS wayland server.'
   homepage 'https://chromium.googlesource.com/chromiumos/containers/sommelier'
-  version '1382ce084cc408'
+  version '1382ce084cc407'
   compatibility 'all'
   source_url 'https://chromium.googlesource.com/chromiumos/containers/sommelier/+/0.20/README?format=TEXT'
   source_sha256 'b58d799b16d20abf92369fe0749c73f7398996f0afa9933517051778a8bb16c3'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/sommelier-1382ce084cc407-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/sommelier-1382ce084cc407-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/sommelier-1382ce084cc407-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/sommelier-1382ce084cc407-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '52c7940172194b786440ea9482835dfdab06df027e91e2e2ca1401acff397e94',
+     armv7l: '52c7940172194b786440ea9482835dfdab06df027e91e2e2ca1401acff397e94',
+       i686: '80c4866b627479b5469cfa04abecc297080b6b330ed9cf8484d2964ed11bfe94',
+     x86_64: '1b33ebc3d0201e1b700b4d55dc53951c77f5f1491af9c24e1c70e66d64107b40',
+  })
 
   depends_on 'mesa'
   depends_on 'xkbcomp'

